lib/server/routes/buckets.js

```diff
@@ -1798,7 +1798,7 @@ BucketsRouter.prototype.finishUpload = async function (req, res, next) {
 
     await Promise.all(bucketEntryShardsToCreate);
 
-    res.status(200).send({ ...savedBucketEntryObject, shards: savedShards });
+    res.status(200).send({ ...savedBucketEntryObject });
```

Comment:
As you are not adding anything to that object anymore, this could be done:

`res.status(200).send(savedBucketEntryObject);`

Reply:
yeah, missed this. Fixed.